<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NcDialog</class>
 <widget class="QDialog" name="NcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网卡设备列表</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>640</width>
     <height>200</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>网卡设备名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>描述信息</string>
    </property>
   </column>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>640</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>您已选择以下设备</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
